src/libs/API/types.ts

```diff
@@ -287,7 +287,7 @@ const READ_COMMANDS = {
     OPEN_WORKSPACE_VIEW: 'OpenWorkspaceView',
     GET_MAPBOX_ACCESS_TOKEN: 'GetMapboxAccessToken',
     OPEN_PAYMENTS_PAGE: 'OpenPaymentsPage',
-    OPEN_PERSONAL_DETAILS_PAGE: 'OpenPersonalDetailsPage',
+    OPEN_PERSONAL_DETAILS: 'OpenPersonalDetailsPage',
```

Comment:
@mountiny - I meant this line, cause key: OpenPersonalDetailsPage should probably be changed to OpenPersonalDetails or something not including 'Page' as there is no longer such page

Reply:
Oh yeah we would have to change that but I think this is not necessary it still makes sense so not worth it